samples/logging/notifications/sample.md

```diff
@@ -35,9 +35,9 @@ snippet: customSLR
 
 ## Plugging to the API
 
-The notifications API is exposed via the BusNotifications class that can be injected via DI.
+The notifications API is exposed via the `BusNotifications` (NServiceBus Version 5) or `Notifications` (NServiceBus Version 6) class that can be injected via DI.
```

Comment:
@mikeminutillo - good point. Here's the mod:
The notifications API is exposed via the BusNotifications class for Versions 5 and below. For Versions 6 and above, the API is available via the Notifications class. This class can be injected [using the container](/nservicebus/containers/.)